README.md

```diff
@@ -18,57 +18,78 @@ LiteFarm is comprised of three applications which all reside in this monorepo.
-   # Install homebrew.
+   In a Terminal window:  
+```      
+   # Install homebrew if you don't already have it with the command:
    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
    # Install PostgreSQL.
    brew install postgresql
-   # Start DBMS service.
+   # Start the Database Management Systems (DBMS) service.
    brew services start postgresql
-   ```
+```
+
+2. Set up the PostgreSQL role (account) and databases. Use the `psql` client program. If an installer asks you to choose a password for the `postgres` (superuser) account, use `postgres` for consistency with the contents of `.env.default`.
 
-2. Set up the PostgreSQL role (account) and databases. You will use the `psql` client program. Account setup details will vary by OS. If an installer asks you to choose a password for the `postgres` (superuser) account, use `postgres` for consistency with the contents of `.env.default`.
+   - On a Mac, type "psql" in the terminal to start the client.
+
+      If this returns the error, "/... postgresql.plist: service already loaded..." then you need to [remove a .pid file](https://stackoverflow.com/questions/13410686/postgres-could-not-connect-to-server) that is interfering with the start of the DBMS service. On a Mac, use the terminal command, `rm /usr/local/var/postgres/postmaster.pid` then `brew services restart postgresql`.
+
+      Then use the Linux commands below to set the postgres user password, and make two new databases.
 
    - Linux. In a terminal, start the client with `sudo -u postgres psql`, then execute each of the following commands. (The last command terminates the client session.)
-       ```sql
+
        ALTER ROLE postgres WITH PASSWORD 'postgres';
        CREATE DATABASE "pg-litefarm";
        CREATE DATABASE test_farm;
+
+   Then exit with,
        exit;       
-       ```
 
    - Windows. At the Start menu, type `psql` and the search results will show "SQL Shell (psql)". In the client, execute
      each of the following commands. (The last command terminates the client session.)
 
-       ```sql
        CREATE DATABASE "pg-litefarm";
        CREATE DATABASE test_farm;
+       
+   Then exit with,
        exit;       
-       ```
+
+   For Windows, the ALTER ROLE command is not used because the password is set using the wizard installer downloaded.
 
 3. In a terminal, navigate to the `packages/api` folder. Execute `npm run migrate:dev:db` to run the [migrations](https://knexjs.org/#Migrations) that set up the PostgreSQL database used by the app.
 
+## Adding environment files
+
+The applications are configured with environment variables stored in `.env` files. Configuration information includes secrets like API keys, so the `.env` files are not included in this git repository.
+
+ This repository only contains `.env.default` files for api and webapp. To join the LiteFarm team and recieve full versions of the environment files contact smattingly@litefarm.org. 
```

Comment:
not sure if smattingly@litefarm.org.  should be mentioned.

Reply:
You mean because his email is there? This line 84 about emailing smattingly@litefarm.org isn't new. It was like that before so I think it should be OK to leave it.

Reply:
Sorry - very late to the party here. We can replace smattingly with kcussen for now.